modules/objdetect/include/opencv2/objdetect/aruco_board.hpp

```diff
@@ -58,14 +58,17 @@ class CV_EXPORTS_W_SIMPLE Board {
     CV_WRAP const Point3f& getRightBottomCorner() const;
 
     /** @brief Given a board configuration and a set of detected markers, returns the corresponding
-     * image points and object points to call solvePnP()
+     * image points and object points, can be used in solvePnP()
      *
      * @param detectedCorners List of detected marker corners of the board.
-     * For CharucoBoard class you can set list of charuco corners.
-     * @param detectedIds List of identifiers for each marker or list of charuco identifiers for each corner.
-     * For CharucoBoard class you can set list of charuco identifiers for each corner.
+     * Board/GridBoard support std::vector<std::vector<Point2f>> or std::vector<Mat> for marker corners.
```

Comment:
For cv::Board and cv::GridBoard the method expects...
For cv::CharucoBoard the method expects std::vector<Point2f> or Mat with ChAruco corners (chess board corners matched with Aruco markers).

Reply:
fixed